src/nets/e2e_tts_th.py

```diff
@@ -30,27 +30,25 @@ def decoder_init(m):
         torch.nn.init.xavier_uniform_(m.weight, torch.nn.init.calculate_gain('tanh'))
 
 
-def make_mask(lengths, dim=None):
-    """FUNCTION TO MAKE BINARY MASK
+def make_non_pad_mask(lengths):
+    """Function to make mask tensor containing indices of non-padded part
 
-    Args:
-        length (list): list of lengths
-        dim (int): # dimension
+    e.g.: lengths = [5, 3, 2]
+          mask = [[1, 1, 1, 1 ,1],
```

Comment:
this example or implementation is wrong. This function returns as same as e2e_asr_th.make_pad_mask
``` python
           mask = [[0, 0, 0, 0, 0], 
                        [0, 0, 0, 1, 1],
                        [0, 0, 1, 1, 1]] 
```
If you really need this consider `~e2e_asr_th.make_pad_mask(length)` instead of redefinition